projects/storefrontlib/src/layout/a11y/keyboard-focus/focus-testing.module.ts

```diff
@@ -5,7 +5,7 @@ import { FocusConfig } from './keyboard-focus.model';
   selector: '[cxFocus]',
 })
 export class MockKeyboardFocusDirective {
-  @Input('cxFocus') protected config: FocusConfig = {};
+  @Input('cxFocus') config: FocusConfig = {};
```

Comment:
Lowering visibility on an existing attribute is a breaking change. 

This looks like a test file, but it does not end with "spec.ts".  I'm not familar with those.

Reply:
since I modified `focus.directive.ts `, so I do the change in it as well.

Reply:
I will revert it back. I found the build error is gone even it is protected :)